gitlab-service-reconection-required.rst

```diff
@@ -0,0 +1,24 @@
+.. post:: Sep 14, 2022
+   :tags: gitlab
+   :author: Santos
+   :location: CUE
+
+GitLab service re-connection required
+=====================================
+
+Some `months ago`_ GitLab started enforcing an expiration time of two hours for all of their OAuth tokens.
+
+.. _months ago: https://gitlab.com/gitlab-org/gitlab/-/blob/master/CHANGELOG.md#1500-2022-05-20
+
+Unfortunately, our application `wasn't ready`_ to handle this change,
+so your OAuth tokens may have expired.
+OAuth tokens are used to interact with the GitLab API,
+for reporting the status of merge requests, creating webhooks, listing repositories etc.
+In order for Read the Docs to have access to new fresh tokens,
+you need to re-connect your GitLab account,
+you can do this at https://readthedocs.org/accounts/gitlab/login/?process=connect,
+or by going to https://readthedocs.org/accounts/social/connections/ and click on ``Connect to GitLab``.
+
+We apologize for any inconvenience caused.
+
+.. _wasn't ready: https://github.com/readthedocs/readthedocs.org/pull/9594
```

Comment:
```suggestion
Unfortunately `this broke`_ the integration with our application,
so your OAuth tokens may have expired.
OAuth tokens are used to interact with the GitLab API,
for reporting the status of merge requests, creating webhooks, and listing repositories.
In order for Read the Docs to have access to new fresh tokens,
you need to re-connect your GitLab account.
You can do this by clicking the `Login with GitLab` button on the homepage of Read the Docs.

We apologize for any inconvenience caused.

.. _this broke: https://github.com/readthedocs/readthedocs.org/pull/9594
```

I removed the links since we can't link directly to both .com & .org

Reply:
I'd use double quotes instead of ticks in "Login with GitLab", from the @ericholscher suggestion.

Reply:
Oh, this won't be fixed by just clicking on login with gitlab (process=login), users need to re-connect their accounts (process=connect).

Reply:
Maybe just put two links? or write the steps of going to your account and click on connec to gitlabt?

Reply:
@stsewd my immediate guess would be that it will look less cluttered if you can write the steps without links, but using abstractions like "In your Read the Docs project dashboard, navigate to `Admin => Integrations => Add Integration` and choose GitLab".